runtime/onert/core/include/ir/train/TrainingInfo.h

```diff
@@ -0,0 +1,68 @@
+
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_TRAIN_TRAINING_INFO_H__
+#define __ONERT_IR_TRAIN_TRAINING_INFO_H__
+
+#include <memory>
+
+#include "ir/train/OptimizerCode.h"
+#include "ir/train/OptimizerInfo.h"
+#include "ir/train/LossInfo.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace train
+{
+
+class TrainingInfo final
```

Comment:
(plan) This TrainingInfo will be saved in nnfw_session : https://github.com/Samsung/ONE/pull/12342/commits/8fa3ffaec360916b60eba426e533da61952fa9e0